### ISSUE TITLE
[Feature Request] HVAC action

### ISSUE THREAD

**Issue by shift-del1** [sockless-coding/panasonic_cc#80]
Can you please add HVAC action (what is the climate doing at the moment, based on the current and the target temperature: heating, cooling, idle) as an attribute? The climate card in UI Lovelace Minimalist currently shows undetermined as this attribute is missing/unavailable.. 

Here is the logic:
```
attributes:
  hvac_action: >
    {% if (is_state('climate.living_room_ac','cool') or is_state('climate.living_room_ac','heat_cool')) and (state_attr('climate.living_room_ac','temperature') < state_attr('climate.living_room_ac','current_temperature')) -%}
      {{ 'cooling' }}
    {%- elif (is_state('climate.living_room_ac','heat') or is_state('climate.living_room_ac','heat_cool')) and (state_attr('climate.living_room_ac','temperature') > state_attr('climate.living_room_ac','current_temperature')) -%}
      {{ 'heating' }}
    {%- else -%}
      {{ 'idle' }}
    {%- endif %}
```

Thank you.

**Comment by sockless-coding** [comment]
This is included in v1.0.33, though the idle state might be incorrectly (not) shown... since the device itself doesn't provide the information of what it's doing.

**Comment by shift-del1** [comment]
> This is included in v1.0.33, though the idle state might be incorrectly (not) shown... since the device itself doesn't provide the information of what it's doing.

You did an excellent job, I did a quick test and tried all HVAC modes and depending on the actual and target temperatures, HVAC action was always correct. I really appreciate your support.
